litmus-portal/frontend/src/pages/LoginPage/index.tsx

```diff
@@ -21,19 +21,13 @@ const LoginPage = () => {
   });
   const [formError, setFormError] = useState<boolean>(false);
 
-  const handleForm = () => {
-    const formData: HTMLFormElement | null = document.querySelector(
-      '#login-form'
-    );
-    const data = new FormData(formData as HTMLFormElement);
-    const username = data.get('username') as string;
-    const password = data.get('password') as string;
-    const searchParams = new URLSearchParams();
-    searchParams.append('username', username);
-    searchParams.append('password', password);
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
     fetch(`${config.auth.url}/login`, {
       method: 'POST',
-      body: searchParams,
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(authData),
     })
       .then((response) => response.json())
```

Comment:
```suggestion
      .then((response) => response.json())
```
Is this line required?

Reply:
More like a sanity check...it helped me debug a lot too...when the server not working due to a small error and the route was not accessible I was not receiving JSON and this will catch that and show an error in the frontend